src/TenantDatabaseManagers/PostgreSQLSchemaManager.php

```diff
@@ -4,18 +4,25 @@
 
 namespace Stancl\Tenancy\TenantDatabaseManagers;
 
+use Illuminate\Database\Connection;
 use Illuminate\Contracts\Config\Repository;
-use Illuminate\Database\DatabaseManager as IlluminateDatabaseManager;
+use Stancl\Tenancy\Contracts\Future\CanSetConnection;
+use Illuminate\Database\DatabaseManager;
 use Stancl\Tenancy\Contracts\TenantDatabaseManager;
 
-class PostgreSQLSchemaManager implements TenantDatabaseManager
+class PostgreSQLSchemaManager implements TenantDatabaseManager, CanSetConnection
 {
-    /** @var \Illuminate\Database\Connection */
+    /** @var Connection */
```

Comment:
To make tenant creation queueable, we can't store the `Connection` instance here, but rather a string and resolve it to a connection using a method call - similarly to https://github.com/stancl/tenancy/blob/2.x/src/TenantDatabaseManagers/MySQLDatabaseManager.php. I'll add this change later today.